templates/azure/shared/_how-ubuntu-pro-helps-you.html

```diff
@@ -1,82 +1,25 @@
-<div class="row">
-  <div class="col-12">
-    <h2>How Ubuntu Pro helps you</h2>
-  </div>
-
-  <div class="col-4">
-    {{ image (
-      url="https://assets.ubuntu.com/v1/ec3f53f7-extended-security-maintenance.svg",
-      alt="",
-      width="60",
-      height="60",
-      hi_def=True,
-      loading="lazy",
-      attrs={"class": "u-hide--small"}
-      ) | safe
-    }}
-
-    <p>Ubuntu Pro enables instant up-to-date security by default with security patches for thousands of open source applications</p>
-  </div>
-
-  <div class="col-4">
-    {{ image (
-      url="https://assets.ubuntu.com/v1/4f70dabc-Kernel.svg",
-      alt="",
-      width="52",
-      height="60",
-      hi_def=True,
-      loading="lazy",
-      attrs={"class": "u-hide--small"}
-      ) | safe
-    }}
-
-    <p>Azure-optimised kernel for improved performance</p>
-  </div>
-
-  <div class="col-4">
-    {{ image (
-      url="https://assets.ubuntu.com/v1/1d5f293c-compliance.svg",
-      alt="",
-      width="105",
-      height="60",
-      hi_def=True,
-      loading="lazy",
-      attrs={"class": "u-hide--small"}
-      ) | safe
-    }}
-
-    <p>Full compliance checklist: FIPS, HIPAA, PCI, ISO and DISA-STIGs</p>
-  </div>
-
-  <div class="col-4">
-    {{ image (
-      url="https://assets.ubuntu.com/v1/b0af9ede-UA_24-7_Support.svg",
-      alt="",
-      width="60",
-      height="60",
-      hi_def=True,
-      loading="lazy",
-      attrs={"class": "u-hide--small"}
-      ) | safe
-    }}
-
-    <p>24/7 support straight from the publisher of Ubuntu*</p>
-
-    <small>* Only available with the private offer</small>
-  </div>
-
-  <div class="col-4">
-    {{ image (
-      url="https://assets.ubuntu.com/v1/ceb481d1-10+years.svg",
-      alt="",
-      width="60",
-      height="60",
-      hi_def=True,
-      loading="lazy",
-      attrs={"class": "u-hide--small"}
-      ) | safe
-    }}
-
-    <p>10 years of extended maintenance</p>
-  </div>
+<div class="col">
+  <h2>How Ubuntu Pro helps you</h2>
+</div>
+<div class="col">
+  <ul class="p-list--divided u-no-margin--bottom">
+    <li class="p-list__item is-ticked" style="box-shadow: unset;">
+      <p class="u-no-padding u-no-margin--bottom">
```

Comment:
Why wrap the content in an additional `p` without any of it's original spacing? Seems like the same thing could be accomplished by not using it at all

Reply:
It still saves some margin there